Wrong import of @formatjs locale-data

These lines references files that doesnt exist.

```
warn ../../node_modules/gatsby-plugin-intl/wrap-page.js
Module not found: Can't resolve '@formatjs/intl-pluralrules/dist/locale-data' in 'web/node_modules/gatsby-plugin-intl'
warn ../../node_modules/gatsby-plugin-intl/wrap-page.js
Module not found: Can't resolve '@formatjs/intl-relativetimeformat/dist/locale-data' in 'web/node_modules/gatsby-plugin-intl'
```
https://github.com/wiziple/gatsby-plugin-intl/blob/ddcc78800008084bdb669831f0f599df6423de5d/src/wrap-page.js#L9-L24

--- a/src/wrap-page.js
+++ b/src/wrap-page.js
@@ -11,12 +11,12 @@ const polyfillIntl = (language) => {
   try {
     if (!Intl.PluralRules) {
       require("@formatjs/intl-pluralrules/polyfill")
-      require(`@formatjs/intl-pluralrules/dist/locale-data/${locale}`)
+      require(`@formatjs/intl-pluralrules/locale-data/${locale}`)
     }
 
     if (!Intl.RelativeTimeFormat) {
       require("@formatjs/intl-relativetimeformat/polyfill")
-      require(`@formatjs/intl-relativetimeformat/dist/locale-data/${locale}`)
+      require(`@formatjs/intl-relativetimeformat/locale-data/${locale}`)
     }
   } catch (e) {
     throw new Error(`Cannot find react-intl/locale-data/${language}`)

